assets/js/modules/pagespeed-insights/components/dashboard/DashboardPageSpeed.js

```diff
@@ -365,9 +372,12 @@ export default function DashboardPageSpeed() {
 				) }
 			>
 				{ dataSrc === DATA_SRC_LAB && (
-					<Link onClick={ updateReport }>
-						{ __( 'Run test again', 'google-site-kit' ) }
-					</Link>
+					<Fragment>
+						<Link onClick={ updateReport } disabled={ isFetching }>
+							{ __( 'Run test again', 'google-site-kit' ) }
+						</Link>
+						<Spinner isSaving={ isFetching } />
+					</Fragment>
```

Comment:
Let's use `<div>` instead of `<Fragment>` to make sure the spinner is shown near to the link. Right now the spinner appears in the center:

![image](https://user-images.githubusercontent.com/873049/147114507-975b26dc-6098-448b-bda5-53e7168a359a.png)